pkg/operator/ceph/nfs/spec.go

```diff
@@ -246,6 +246,17 @@ func (r *ReconcileCephNFS) daemonContainer(nfs *cephv1.CephNFS, cfg daemonConfig
 		Env:             controller.DaemonEnvVars(r.cephClusterSpec),
 		Resources:       nfs.Spec.Server.Resources,
 		SecurityContext: controller.PodSecurityContext(),
+		LivenessProbe: &v1.Probe{
```

Comment:
```suggestion
		LivenessProbe: &v1.Probe{
```

I think we need to make it configurable via crd. You can refer to cluster.yaml  and look at other examples we have for osd, mons... https://github.com/rook/rook/blob/ded675e758bd8870d0b32fe882b035582b91789e/pkg/operator/ceph/cluster/mgr/spec.go#L193

Reply:
Fair enough -- I will fix accordingly.